src/lib.rs

```diff
@@ -42,6 +42,30 @@
 use std::collections::HashMap;
 use testcontainers::{core::WaitFor, Container, Image};
 
+#[derive(Clone, Copy, Debug)]
+pub enum Neo4jLabsPlugin {
+    Apoc,
+    ApocCore,
+    Bloom,
+    Streams,
+    GraphDataScience,
+    NeoSemantics,
+}
+
+impl Neo4jLabsPlugin {
+
+    fn get_value(&self) -> String {
+        match self {
+            Neo4jLabsPlugin::Apoc => {"apoc"}
+            Neo4jLabsPlugin::ApocCore => {"apoc-core"}
+            Neo4jLabsPlugin::Bloom => {"bloom"}
+            Neo4jLabsPlugin::Streams => {"streams"}
+            Neo4jLabsPlugin::GraphDataScience => {"graph-data-science"}
+            Neo4jLabsPlugin::NeoSemantics => {"n10s"}
+        }.to_owned()
+    }
+}
```

Comment:
I would probably implement the [`Display`](http://doc.rust-lang.org/stable/std/fmt/trait.Display.html) instead. Every type that implements `Display` also implements `ToString`, so we can use `to_string()` if we want the value as a `String`.

```suggestion
impl std::fmt::Display for Neo4jLabsPlugin {
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
        match self {
            Self::Apoc => f.pad("apoc"),
            Self::ApocCore => f.pad("apoc-core"),
            Self::Bloom => f.pad("bloom"),
            Self::Streams => f.pad("streams"),
            Self::GraphDataScience => f.pad("graph-data-science"),
            Self::NeoSemantics => f.pad("n10s"),
            // If you add the Custom variant, you would use it like this
            Self::Custom(plugin) => f.pad(plugin),
        }
    }
}
```

This allows us to use the type directly in `{}` directives in format strings.